kivy/__init__.py

```diff
@@ -481,7 +481,8 @@ def _patch_mod_deps_win(dep_mod, mod_name):
     if need_save and 'KIVY_NO_CONFIG' not in environ:
         try:
             with open(kivy_config_fn, 'w') as fd:
```

Comment:
The file gets opened with the proper permissions and encoding into the `write` method, so no need to open it.

Reply:
I'm sorry for my mistake! I've fixed it.